app/core/Tracker2/GoogleAnalyticsTracker.js

```diff
@@ -40,8 +40,8 @@ export default class GoogleAnalyticsTracker extends BaseTracker {
     if (properties.label) {
       gaFieldObject.eventLabel = properties.label
     }
-    if (properties.value) {
-      gaFieldObject.eventValue = properties.value
+    if (properties.value || properties.predictedLtv) {
+      gaFieldObject.eventValue = properties.value || properties.predictedLtv
     }
```

Comment:
I think we also need to set `eventValue` as `properties.purchaseAmount` if we don't have a  `predictedLtv` (like for a one-time purchase)?